pkg/v1/cli/catalog/catalog.go

```diff
@@ -0,0 +1,227 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package catalog
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	apimachineryjson "k8s.io/apimachinery/pkg/runtime/serializer/json"
+
+	cliv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/cli/v1alpha1"
+	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/cli/common"
+	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/utils"
+)
+
+const (
+	// catalogCacheFileName is the name of the file which holds Catalog cache
+	// TODO: Use the original catalog file instead of using v2 once the feature is enabled by default
```

Comment:
**thought**: This seems like an easy thing to forget to do. Is there some way to more clearly associate it with making the feature enabled by default? (E.g., perhaps leaving a comment over in clientconfig.go referencing this file? Or extracting the default `isContextAwareDiscoveryEnabled` into a const and adding a little unit test that fails if that const becomes `true` and this `catalogCacheFileName` const hasn't been updated?)

Reply:
Addressed by adding constant and a unit test.